test/e2e/cli/cli_test.go

```diff
@@ -586,6 +589,70 @@ var _ = Describe("cli operation", func() {
 			out, err = harness.RunGetEvents(limit, "1", "2")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring("accepts 1 arg(s), received 2"))
+
+			By("fetching the next page of events using the continue flag", func() {
+				// First, get the first page with limit 1
+				out, err := harness.RunGetEvents(limit, "1", jsonFlag)
+				Expect(err).ToNot(HaveOccurred())
+
+				var firstPage struct {
```

Comment:
You can use Event struct, I believe

Reply:
Event struct probably won't work here as it is yaml output. Please, create support function as Silvia suggested (you define the events 3 times here